tidy/lib.py

```diff
@@ -59,17 +71,15 @@ def __init__(self, libnames: Optional[Tuple[str, ...]] = None):
                 break
             except OSError:
                 continue
-
-        # Fail in case we could not load it
-        if self.lib is None and "IGNORE_MISSING_TIDY" not in os.environ:
+        else:
```

Comment:
Why remove `IGNORE_MISSING_TIDY` path? It is used to build the documentation. In that case, the library is not really working, but only imported for the autodoc purposes. There might be better ways to achieve that, but this is what is used right now.

Reply:
Okay, now it is possible to install library in readthedocs and I've merged that in https://github.com/nijel/utidylib/pull/109 so this path can now be removed.